test/azure/legacy/specification/packagemodebatch/README.md

```diff
@@ -0,0 +1,41 @@
+# Testing package-mode
+
+### Settings
+
+``` yaml
+package-version: 1.0.0
+license-header: MICROSOFT_MIT_NO_VERSION
+output-artifact: code-model-v4-no-tags
+payload-flattening-threshold: 1
+package-mode: dataplane
+package-name: azure-packagemode-batch
+package-pprint-name: Azure Package Mode Batch Data Plane
+output-folder: $(python-sdks-folder)/azure/legacy/Expected/AcceptanceTests/PackageModeBatch
+clear-output-folder: true
+```
+
+### Python multi-client
+
+``` yaml
+batch:
+    - tag: head
+    - tag: paging
+    - multiclientscript: true
+```
+
+``` yaml $(tag) == 'head'
+title: HeadClient
+input-file: ../../../../../node_modules/@microsoft.azure/autorest.testserver/swagger/head.json
+namespace: azure.packagemode.batch.head
+```
+
+``` yaml $(tag) == 'paging'
+title: PagingClient
+input-file: ../../../../../node_modules/@microsoft.azure/autorest.testserver/swagger/paging.json
+namespace: azure.packagemode.batch.paging
+```
+
+``` yaml $(multiclientscript)
```

Comment:
like the readme. Can you also add an acceptance test to make sure we can install it, and at least initialize clients (and ideally make a call or two)? Thanks! Otherwise looks good!

Reply:
added: test_multi_client.py